14th-team5-iOS/Core/Sources/Bibbi/BBCommons/BBView/BBEmptyView.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  BBEmptyView.swift
+//  Core
+//
+//  Created by 마경미 on 15.04.25.
+//
+
+import UIKit
+
+import DesignSystem
+
+final public class BBEmptyView: UIView {
```

Comment:
```suggestion
public final class BBEmptyView: UIView {
```
이렇게 하면 좋을 것 같은데 어떠신가용??